tests/acceptance/features/apiGraph/editGroup.feature

```diff
@@ -22,4 +22,22 @@ Feature: edit group name
       | grp1      | नेपाली          |
       | grp1      | $x<=>[y*z^2]! |
       | grp1      | staff?group   |
-      | grp1      | 50%pass       |
\ No newline at end of file
+      | grp1      | 50%pass       |
+
+
+  Scenario Outline: user other than the admin can't rename a group
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And group "grp1" has been created
+    When user "Alice" tries to rename group "grp1" to "grp101" using the Graph API
+    Then the HTTP status code should be "401"
+    Examples:
+      | role        |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+
+  Scenario: admin user tries to rename non-existent group
+    When user "Alice" tries to rename non-existent group to "grp1" using the Graph API
```

Comment:
```suggestion
  Scenario: admin user tries to rename nonexistent group
    When user "Alice" tries to rename nonexistent group to "grp1" using the Graph API
```

Reply:
Yeah, you can do it quickly, just two lines. Also, you have done similar in other places as well